### ISSUE TITLE
Pls check the letter-spacing for Chinese in Editing Mode

### ISSUE THREAD

**Issue by cositan4lrb** [HappenApps/Quiver#312]
![image](https://cloud.githubusercontent.com/assets/8206111/11767371/b91bb034-a168-11e5-8166-9a4ce3bb76b9.png)

You can see that it's very weird that the Chinese characters have some little spacing between each other.

Then I check the css file for editing mode. And I add some commands to set the letter-spacing, but I found when the value is set "normal", the English letters are really normal, the Chinese characters have some weird spaces.

Compared to "Preview Mode", the unchanged ("normal") letter-spacing, the issue is fixed this time.

So I really doubt there would be another css file control the Editing Mode Markdown part.

This is result from another Markdown product named Ulysses.

![image](https://cloud.githubusercontent.com/assets/8206111/11767408/b41bf3ae-a169-11e5-9944-53ec7ae67bf6.png)

Pls double check the issue. I love your product.

**Comment by ylian** [comment]
Thanks. The Ace editor doesn't support multibyte languages very well: https://github.com/ajaxorg/ace/issues/460

But some user finds that setting the font-family to SimSun fixes the issue. In Preferences > Styles > Editor > Edit CSS, change Markdown cell settings to:

``` css
.markdown-cell .ace_editor {
  background-color: #FFFFFF;
  color: #333;
  font-size: 12px;
  font-family: SimSun;
  line-height: 1.5;
}
```

**Comment by ylian** [comment]
Wow, obviously there is another fix (from https://twitter.com/Fate_Stigma/status/676050550069272577).

Set this in the Markdown cell:

``` css
.markdown-cell .ace_cjk {
    width: 12px !important;
}
```

I will check if this fix has any side effects. If not, I will include this fix in the default CSS.

**Comment by cositan4lrb** [comment]
Actually, this issue has been fixed. But it causes another issue.

![image](https://cloud.githubusercontent.com/assets/8206111/11769609/f0e77490-a1a8-11e5-8420-b38d2bdd75fa.png)

You can see the cursor has some position problems --- it just follows the previous spacing setting.

**Comment by ylian** [comment]
Yep your are right. Try the first method. It seems to work for me without causing the cursor positioning issue.

``` css
.markdown-cell .ace_editor {
  background-color: #FFFFFF;
  color: #333;
  font-size: 12px;
  font-family: SimSun;
  line-height: 1.5;
}
```

**Comment by cositan4lrb** [comment]
I don't know why. It does not work for me. The first method could not fix the first problem --- the weird spacing.

**Comment by ylian** [comment]
It does seem to work for me:

![screen shot 2015-12-13 at 6 48 29 pm](https://cloud.githubusercontent.com/assets/984895/11770362/2e5a9b88-a1ca-11e5-9c9a-e17e3863e08d.png)

Check your editor CSS. There was already a css rule for `.markdown-cell .ace_editor`. You might have copied in the above snippet, but didn't delete the existing rule.

**Comment by herrkaefer** [comment]
Hi @ylian , I love this app except the weird spacing between Chinese characters. The first method does not work for me either. Would you please recheck it ?

**Comment by ylian** [comment]
@herrkaefer The first method does work for me. I don't know why. If not, the real fix would require some significant work in the Ace editor, and I won't be able to do that in the near future.

**Comment by herrkaefer** [comment]
@ylian @cositan4lrb I got a solution by setting a fixed-width font which also supports Chinese language. I use Google's [Noto Sans Mono CJK](https://www.google.com/get/noto/help/cjk/)

``` css
.markdown-cell .ace_editor {
  background-color: #FFFFFF;
  color: #333;
  font-size: 15px;
  font-family: NotoSansMonoCJKsc-Regular;
  line-height: 1.5;
}
```

which works perfect for me. No cursor problem.

**Comment by ylian** [comment]
Merged into #432.
